provider/adapters/src/fallback/algorithms.rs

```diff
@@ -202,6 +212,7 @@ mod tests {
         input: &'static str,
         requires_data: bool,
         extension_key: Option<Key>,
+        fallback_supplement_key: Option<DataKey>,
```

Comment:
This is always `None`. Either remove or have some `Some` test.

Reply:
There is a docs test for it attached to `LocaleFallbackConfig::fallback_supplement_key`